pkg/kubectl/cmd/run.go

```diff
@@ -478,8 +478,9 @@ func handleAttachPod(f *cmdutil.Factory, c *client.Client, ns, name string, opts
 	opts.Client = c
 	opts.PodName = name
 	opts.Namespace = ns
+	stderr := opts.Err
 	if err := opts.Run(); err != nil {
-		fmt.Fprintf(opts.Err, "Error attaching, falling back to logs: %v\n", err)
+		fmt.Fprintf(stderr, "Error attaching, falling back to logs: %v\n", err)
```

Comment:
@fraenkel maybe add a TODO or comment here mentioning that opts.Run() sets opts.Err to nil and we should find a better way to do that